### PR TITLE
#2843 [Manager] Use cluster keys instead of cluster names

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2843)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/cypress/support/apiCommands.js [hunk 1]
```diff
@@ -319,4 +319,13 @@ Cypress.Commands.add('deleteAllServices', () => {
       });
     }
   });
+
```
Comment (konekoya) [inline ohara-manager/client/cypress/support/apiCommands.js]: Clean nodes as well

File: ohara-manager/client/cypress/e2e/streamApp.test.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ describe.skip('StreamApp', () => {
       prefix: 'pipeline',
       length: 5,
     });
```
Comment (konekoya) [inline ohara-manager/client/cypress/e2e/streamApp.test.js]: The name could be too long to be selected at as we truncate text that's long enough to break into new line

File: ohara-manager/client/cypress/api/brokerApi.test.js [hunk 3]
```diff
@@ -44,7 +44,10 @@ const setup = () => {
   cy.createBroker({
     name: brokerClusterName,
     nodeNames: [nodeName],
-    zookeeperClusterName,
+    zookeeperClusterKey: {
```
Comment (konekoya) [inline ohara-manager/client/cypress/api/brokerApi.test.js]: Use zookeeper and broker cluster key and

File: ohara-manager/client/cypress/e2e/streamApp.test.js [hunk 4]
```diff
@@ -85,6 +85,8 @@ describe.skip('StreamApp', () => {
       .click()
       .wait('@putPipeline');
```
Comment (konekoya) [inline ohara-manager/client/cypress/e2e/streamApp.test.js]: After we migrated our codebase to use hooks, it sometimes makes more http requests then we anticipated. I don't have a good solution for now (and not sure if it's the real cause)

File: ohara-manager/client/src/components/pages/PipelinePage/Connectors/StreamApp.js [hunk 5]
```diff
@@ -348,7 +370,7 @@ StreamApp.propTypes = {
     ).isRequired,
     tags: PropTypes.shape({
       workerClusterName: PropTypes.string.isRequired,
-      brokerCluterName: PropTypes.string.isRequired,
+      brokerClusterName: PropTypes.string.isRequired,
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/PipelinePage/Connectors/StreamApp.js]: Add a broker name here as we now need to include the broker cluster name when updating a strem app

**Comment by konekoya** [comment]
This issue also addresses #2731

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by saivirtue** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by jackyoh** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Pull request merged**